docs/guide/guides/push-notifications-firebase.md

```diff
@@ -379,7 +381,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   }
 ```
 
-If you would like to recieve the firebase FCM token from iOS instead of the raw APNS token, you will need to also change your `AppDelegate.didRegisterForRemoteNotificationsWithDeviceToken` code to look like this:
+If you would like to recieve the firebase FCM token from iOS instead of the raw APNS token, you will need also add two new `import` at the top of the file :
```

Comment:
```suggestion
If you would like to receive the Firebase FCM token from iOS instead of the raw APNS token, you will also need to add two new imports at the top of the file:
```